t/ui/12-needle-edit.t

```diff
@@ -96,22 +96,15 @@ shared_hash {};
 my $git_mock = Test::MockModule->new('OpenQA::Git');
 $git_mock->redefine(commit => sub ($self, $args) { shared_hash $args; return undef });
 
-# prepare clean needles directory
-my $dir = path('t/data/openqa/share/tests/opensuse/needles')->remove_tree->make_path;
-
 plan skip_all => $OpenQA::SeleniumTest::drivermissing unless my $driver = call_driver({with_gru => 1});
 
+# prepare clean needles directory, create default 'inst-timezone' needle
+my $dir              = prepare_clean_needles_dir;
```

Comment:
Maybe use the name `$needle_dir` here as well analog to `t/ui/18-tests-details.t`